### PR TITLE
Feat/met 1820 remove plugins from core mongo

### PR DESCRIPTION
**Pull request by jochen-vermeulen**

### CODE REVIEW

File: remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  private static List<PluginToRemove> readFile() throws IOException {
+    final List<PluginToRemove> result = new ArrayList<>();
+    final Path path = Paths.get(FILE_FOR_PLUGIN_REMOVAL);
+    try (final BufferedReader fileReader = Files.newBufferedReader(path, StandardCharsets.UTF_8)) {
+      final CSVReader reader = new CSVReader(fileReader);
+      LOGGER.warn("Skipping first line: [{}].", String.join(", ", reader.readNext()));
+      for (String[] line : reader) {
+        if (line.length != 3) {
+          throw new IllegalStateException(
+              "Problem in line " + result.size() + " with data [" + String.join(", ", line)
```
Comment (stzanakis) [inline remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java]: We try to use String.format(), which makes the messages a bit cleaner to read.

File: remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+  private static final String FILE_FOR_PLUGIN_REMOVAL = "/home/jochen/Desktop/plugins_to_remove.csv";
+
+  public static void main(String[] args) throws TrustStoreConfigurationException, IOException {
+    try (final Application application = Application.initialize()) {
+      final List<PluginToRemove> plugins = readFile();
+      LOGGER.info("Removing {} plugins.", plugins.size());
+      final WorkflowExecutionDao dao = new WorkflowExecutionDao(application.getDatastoreProvider());
+      plugins.stream().map(plugin -> RemovePluginsMain.findPlugin(plugin, dao))
+          .filter(Objects::nonNull).filter(plugin -> canEditWorkflow(plugin.getLeft()))
```
Comment (stzanakis) [inline remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java]: The parameter "plugin" in the lambdas is a pair of execution and plugin, perhaps rename it?

File: remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java (hunk 1 above)
Comment (jochen-vermeulen) [reply remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java]: Fixed.

File: remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java (hunk 2 above)
Comment (jochen-vermeulen) [reply remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java]: Fixed.

**Pull request merged**